if-else-js.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>if-else</title>
+</head>
+<body>
+<script
+    src="js/if-else.js">
```

Comment:
You are missing another attribute here. I'm sure you know what is missing but here ya [go](https://www.w3schools.com/tags/att_script_type.asp)